### ISSUE TITLE
About loss_function

### ISSUE THREAD

**Issue by Seconight** [Yanfeng-Zhou/XNet#10]
Hi! Thank you for your great work! 🤗
I have a question when I train the 3D model on the Atrial dataset with the script `train_sup_XNet3d.py`.
There was an exception:
![image](https://github.com/Yanfeng-Zhou/XNet/assets/54622036/438833ed-4cea-4dd7-beeb-8d1bba3fadd1)
Then I checked the corresponding tensors in `loss_function.py`. 
For `forward(self, output, target)` in `class DiceLoss(nn.Module)` :
```Python
def forward(self, output, target):
        # preds, target = tuple(inputs)
        # inputs = tuple(list(preds) + [target])
        if self.aux:
            return self._aux_forward(output, target)
        else:
            valid_mask = (target != self.ignore_index).long()
            target_one_hot = F.one_hot(torch.clamp_min(target, 0))
            
            return self._base_forward(output, target_one_hot, valid_mask)
```
The shape of `output` is torch.Size([1, 2, 96, 96, 80]), however the shape of `target_one_hot` is torch.Size([1, 96, 96, 80, 256]). There is only 0 and 255 in target, should the target be 1?

**Comment by Yanfeng-Zhou** [comment]
In order to facilitate the visualization of segmentation masks, the foreground in the dataset is generally marked as **255**. During training, we should convert **255** to **1**, that is, **foreground = 1, background = 0**.
